lineman/app/components/inbox_page/inbox_page_controller.coffee

```diff
@@ -1,17 +1,23 @@
-angular.module('loomioApp').controller 'InboxPageController', ($rootScope, Records, CurrentUser, LoadingService, ThreadQueryService) ->
+angular.module('loomioApp').controller 'InboxPageController', ($scope, $rootScope, Records, CurrentUser, LoadingService, ThreadQueryService) ->
   $rootScope.$broadcast('currentComponent', {page: 'inboxPage'})
   $rootScope.$broadcast('setTitle', 'Inbox')
 
   @threadLimit = 5
 
-  _.each CurrentUser.parentGroups(), (group) =>
-    @["group#{group.id}"] = ThreadQueryService.groupQuery(group)
-  @baseQuery = ThreadQueryService.filterQuery('show_unread')
+  @groups = ->
+    CurrentUser.parentGroups()
 
-  @queryFor = (group) -> @["group#{group.id}"]
+  @init = =>
+    _.each @groups(), (group) =>
+      @["group#{group.id}"] = ThreadQueryService.groupQuery(group)
+    @baseQuery = ThreadQueryService.filterQuery('show_unread')
+  @init()
+  $scope.$on 'currentUserMembershipsLoaded', @init
```

Comment:
Since we kicked off the memberships into a separate request, we have to wait for that to come back before we set up all the group queries.